Readme.md

```diff
@@ -78,6 +78,20 @@ terraform init -input=false \
   -backend-config="storage_account_name=$saName" \
   -backend-config="container_name=$scName"
 ```
+## Azuread provider configuration
+```
+terraform {
+  required_providers {
+    azuread = {
+      source  = "hashicorp/azuread"
+      version = "=1.5.0"
```

Comment:
version = ">=1.5.0" - not production ready but helps users to get started with the newest version